### DISCUSSION TITLE
[Weekly Curation Report]: October 9 - 15

### DISCUSSION THREAD

**Discussion by danidi** [wikipathways/wikipathways-help#74] [curation-reports]
### Recent changes

Recent Changes: 
22 pathways with changes.
@khanspers, I've seen several changes adding phosphosites, but I couldn't see what the actual changes were. There are no comments on phosphorylation added on the data nodes.

@JuliaBatUM reverted a change by @egonw  (fixing the Ensembl datasource label) on https://classic.wikipathways.org/index.php/Pathway:WP5416. Was this done on purpose? Ensembl Human does not work with the linkouts, Ensembl should be the correct label.

There were some small changes to quite old non-approved pathways. Maybe worthwhile to improve and appove before at some point?
https://classic.wikipathways.org/index.php/Pathway:WP4281 (Cholesterol biosynthesis, regulation and transport): very detailed description, but no references. I've added the needs work tag to find it more easily. 
https://classic.wikipathways.org/index.php/Pathway:WP4359 (Splisiosome)

### Under construction

Under Construction: 
Removed the under construction tag from the following pathways (no changes in the last 30 days), where not present I've added a needs work tag:
https://classic.wikipathways.org/index.php/Pathway:WP5361
https://classic.wikipathways.org/index.php/Pathway:WP5386
https://classic.wikipathways.org/index.php/Pathway:WP5410

https://classic.wikipathways.org/index.php/Pathway:WP5411 looks quite complete to me, only the layout looks a bit confusing. I've added the approved tag.

### New pathways

New pathways: 
Synthesis of acylcarnitines: https://classic.wikipathways.org/index.php/Pathway:WP5423

### Deleted Pathways

Pathways for deletion: 
https://classic.wikipathways.org/index.php/Pathway:WP5415 should be deleted, but even though I could usually perform deletions, I don't see the option anymore. 

Requested deletion for https://classic.wikipathways.org/index.php/Pathway:WP5350, @khanspers please remove the tag if you still need the pathway. 

### Additional notes

The note on the first page of the QA protocol (suggestion to update the approved tags several times per week) was read too late...

@egonw noticed https://www.wikipathways.org//pathways/WP1028.html is a homology conversion of WP134, but does not have some changes made to the original human pathway since then. (The description is missing and some lines are unconnected). Maybe removing the curated tag would be best. 

https://classic.wikipathways.org/index.php/Pathway:WP4234 is an annotated list, but not approved. What will happen with those pathways once classic is no longer available? Adding @DeniseSl22 who created the pathway initially.

Created issues/discussions: 
https://github.com/wikipathways/wikipathways-help/discussions/73

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

**Comment by khanspers** [comment]
Thanks for the great report @danidi! For the phosphosite updates to several pathways, they won't be visible on the website (classic or new) because the structured comment is added to the State node, which isn't listed in the DataNodes table. The changes I made were to update the structured comment to the new format, which includes additional information such as site sequence and site group id. I also removed and added some sites reflecting new information. 
I have deleted WP5415, and updated your permissions so you shouldn't have a problem deleting pathways now. I also deleted WP5350.

**Comment by egonw** [comment]
> they won't be visible on the website (classic or new) because the structured comment is added to the State node, which isn't listed in the DataNodes table

Shall we make this two "enhancements"? If you let me know how the comment is structured, I could see if I can do something with that in the RDF. Second, one the new website we should do something with the `State` info, anyway.

**Comment by JuliaBatUM** [comment]
Dear @danidi ,

I unintentionally reverted the changes made by @egonw, and I deeply apologize for this mistake.

I was wondering if it is possible to pinpoint the specific locations in the map where these changes were initially implemented? 

Furthermore, I would like to know if I can delete my pathway, WP5416. I have made significant alterations to this pathway, but unfortunately, I was now aware that these changes must be performed within PathVisio via the plugin-> wikipathways->Search method. Consequently, I was unable to update the pre-existing version of WP5416. To save time, I decided to create a new pathway, WP5424, to incorporate all the new information I have gathered. From now on, I will update pathways as recommended via the plugin-> wikipathways->Search method to avoid any further issues.

Thank you for your understanding.

Sincerely,

Julia

**Reply by mkutmon** [reply]
@JuliaBatUM - finding the differences will be much easier in the new system. On the old site, we only have the "Compare selected versions" which is not always useful. 
I will delete the old pathway. Next time, feel free to contact us! There are some workarounds that we can use to update such pathways :).

**Reply by danidi** [reply]
Thank you for the clarification, @JuliaBatUM! By the way, WP5424 contains a strange identifier "j (Ensembl Human)" for an instance of the Rb gene product.
